addons/pronelauncher/CfgMovesBasic.hpp

```diff
@@ -0,0 +1,123 @@
+class CfgMovesBasic {
+    class Default;    // External class reference
+    class NoActions; // External class reference
+    class Actions {
+        class CivilStandActions;    // External class reference
+        class CivilProneActions : CivilStandActions {
+            SecondaryWeapon = "ACE_LauncherProne";
```

Comment:
maybe all these `ACE_` names should be replaced with GVAR/QGVAR values

Reply:
https://github.com/acemod/ACE3/blob/e5a15d200f44df5fccc0bc5575d18d80b35538dd/addons/movement/CfgMoves.hpp#L102

Reply:
As author of the ACE_Climb class, this is fine :P